packages/odyssey-react-mui/src/Dialog.tsx

```diff
@@ -139,9 +139,17 @@ const Dialog = ({
   ].filter(Boolean);
 
   return (
-    <MuiDialog data-se={testId} open={isOpen} onClose={onClose}>
-      <DialogTitle translate={translate}>
-        {title}
+    <MuiDialog
+      data-se={testId}
+      open={isOpen}
+      onClose={onClose}
+      aria-labelledby="ods-dialog-title"
+    >
+      <DialogTitle
+        id="ods-dialog-title__container" // We need to explicitly unset `id` for MUI to automatically set it based on the `aria-labelledby` prop passed to `MuiDialog`
```

Comment:
I'm not sure if we can have multiple dialogs on the page at once. If not, **NON-BLOCKING**.

If we can, this will fail. We'd need to add a postfix of Odyssey's `useUniqueId` or simply just use the generated `id` altogether. We don't want people targeting this `id` anyway.

Reply:
updated